cypress/fixtures/docs/validations.md

```diff
@@ -0,0 +1,615 @@
+# Supported Validations
+
+| **Validation Type** | **Description**                                                                                               |
```

Comment:
where have you provided the link to this doc ?

Reply:
added this doc in static JS, static JSON and dynamic readme